pkg/subctl/cmd/gather.go

```diff
@@ -148,6 +170,8 @@ func gatherConnectivity(dataType string, info gather.Info) bool {
 		gather.RouteAgentPodLogs(info)
 		gather.GlobalnetPodLogs(info)
 		gather.NetworkPluginSyncerPodLogs(info)
+		gather.CNILogs(info)
```

Comment:
I would consider all this resources instead, it's technically not logs.

Can we call them resources instead? :)

Reply:
I used logs because `resources` is a very k8s term that refers to `api-resources` or `CustomResource`